pages/TheExplorationGallery.html

```diff
@@ -11,32 +11,39 @@
   <p>
     After signing in to Oppia, you'll end up on the Home page. Even before you
```

Comment:
Actually, this is no longer true, isn't it? Now, after signing in, you end up on the dashboard.

So, maybe omit the 'sign-in' part, and instead start with: The Library page is the index page to access explorations that others have created. You can get to it by clicking the 'Library' tab at the top-right corner of the page, or, if your screen is small, click the 'hamburger' menu at the top-left and then select 'Library' from the list of options that appears.

(Then, follow this by a single image, containing two screenshots, side by side, that show the two possible cases. In each case, an arrow + circle points to the thing to be clicked.)